field_test.go

```diff
@@ -166,3 +182,17 @@ func TestFieldInlineComment(t *testing.T) {
 		t.Error("expected inline comment")
 	}
 }
+
+func TestFieldTypeStartsWithDot(t *testing.T) {
```

Comment:
Is there an integration test we can add that parses an entire Protobuf file into a `*proto.Proto`, for example it would parse this:

```proto
syntax = "proto3";

package test;

message Foo {
  .game.Resource one = 1 [deprecated = true];
  repeated .game.sub.Resource two = 2;
  map<string, .game.Resource> three = 3;
}
```